cloud-sql/mysql/sqlalchemy/main.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import logging
+import os
+
+from flask import Flask, render_template, request, Response
+import sqlalchemy
+
+
+# Saving credentials in environment variables is convent, but not secure
```

Comment:
Ditto. This isn't the worst practice, and might actually offend the 12-factor app developer community. This comment _is_ appropriate in the `app.yaml` file, though. You don't want people checking secrets into version control.